src/qibolab/instruments/zhinst.py

```diff
@@ -814,6 +816,8 @@ def sweep_recursion(self, qubits, exp, exp_calib, relaxation_time, acquisition_t
                     )
                 )
 
+            print(ZhSweeper(pulse, sweeper, qubits[sweeper.pulses[0].qubit]).zhsweeper)
```

Comment:
I guess this is here to debug, but since Qibolab is a library, we should eventually avoid printing (at most you can log, but only if very relevant).

Make sure to remove all prints before merging.